file_test.go

```diff
@@ -30,6 +30,8 @@ import (
 	"time"
 
 	"github.com/moov-io/base"
+
+	"github.com/stretchr/testify/require"
```

Comment:
Sweet, I was hesitant about using `testify` in this repo, but I'll start using it too

Reply:
Yea. I figure it's good enough. It'll only be included in tests also.